tests/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+include(FetchContent)
+FetchContent_Declare(googletest 
+    GIT_REPOSITORY 
+        "https://github.com/google/googletest.git"
+    GIT_TAG
+        release-1.12.0
+)
+FetchContent_GetProperties(googletest)
+if(NOT googletest_POPULATED)
+    message(STATUS "Fetching gtest and gmock from github.com")
+    FetchContent_Populate(googletest)
+    add_subdirectory(${googletest_SOURCE_DIR} ${CMAKE_BINARY_DIR}/_deps/googletest)
+endif()
```

Comment:
I also used to do that, but I for libraries I would prefer not to use fetchcontent.  For now you could also just use the dependencies.yaml

Reply:
Is there any reason why not simply ask the friendly local package manager to install that gtest stuff?